### ISSUE TITLE
Ranking model predicting same value for all

### ISSUE THREAD

**Issue by harry7171** [tensorflow/recommenders#377]
Hi,

I am training a Ranking model with some basic numeric features and the target is either 0 or some value between 0-1. 
So the model is training and just in 1 epoch it comes to a minimum loss and doesnot decrease more.
The main issue is observed when i predict with test data , i get all the predictions as 0. tried some changes in Layers but all leads to almost 99% values as 0 
Where as same architecture when i trained using sequential model . its learning and getting atleast some values in prediction .
Not sure why with tfrs Ranking its some issue .

**Comment by maciejkula** [comment]
There is not a whole lot that I can say based on this information.

There could be something wrong with your data, or your loss, or your model architecture, or your training hyperparameters.
